testdata/bless_provider/scripts/component.mk

```diff
@@ -8,41 +8,71 @@ include $(SELF_DIR)/common.mk
 all:
 .PHONY: all
 
+setup:
+	$(MAKE) -C $(REPO_ROOT) setup
```

Comment:
does this need a .PHONY?

Reply:
yeah, will get generated by fixiing above